iidm/iidm-api/src/test/java/com/powsybl/iidm/network/util/BranchTestData.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.util;
+
+import org.mockito.Mockito;
+
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.Line;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.iidm.network.VoltageLevel;
+import com.powsybl.iidm.network.Terminal.BusView;
+
+/**
+ *
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class BranchTestData {
```

Comment:
This class, its constructor and its methods can be private-package

Reply:
Done